now.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This file is specific for [now.sh](https://zeit.co/) deploys. I know there's a description on the **README** about this I don't think this should come in the boilerplate.

Also, using version **2** means that we loose SSR since the app will be served as serverless.

Reply:
-- We should analyse other alternatives like Netlify, but for now we should keep this since it is currently our way of staging.

Reply:
Opened issue. I will keep `now` for now since it is our bet at the time.